learn/DOM/004_Methods/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DOM Methods</title>
+</head>
+<style>
+    div {
+        border: 2px solid blue;
+        margin: 10px auto;
+        text-align: center;
+        padding: 10px;
+    }
+    .gradient{
+        background: linear-gradient(to bottom, blue , yellow , red );
+        border-radius: 10px;
+        border-color: transparent;
+    }
+</style>
+
```

Comment:
Harusnya susunan htmlnya kayak gini, terus juga karena di file js ada syntax `console.log`, biar ga ribet nanti di #171, langsung tambahin aja fungsi yang ngirim pesan ke parent

```suggestion
   <script>
      const _log = console.log;
      console.log = function (...rest) {
        window.parent.postMessage(
          {
            source: "iframe",
            message: rest
          },
          "*"
        );
        _log.apply(console, arguments);
      };
    </script>
    
    <style>
        div {
            border: 2px solid blue;
            margin: 10px auto;
            text-align: center;
            padding: 10px;
        }
        .gradient{
            background: linear-gradient(to bottom, blue , yellow , red );
            border-radius: 10px;
            border-color: transparent;
        }
    </style>

    <title>DOM Methods</title>
</head>
```

Reply:
apa di file javascriptnya say hapus console.lognya

Reply:
`console.log` mah biarin aja, tambahin yang ini aja

Reply:
java script di atas tag style udah di tambahin

Reply:
![bandicam 2021-11-03 13-21-24-817](https://user-images.githubusercontent.com/27930710/140013244-294547b7-d435-4e83-856d-4ced37dd123e.jpg)
sudah di tambahkan text javascriptnya ya om